include/loops/loops.hpp

```diff
@@ -605,6 +612,10 @@ template<typename _Tp> VReg<_Tp> operator - (const VReg<_Tp>& a)
 { return newiopV<_Tp>(VOP_NEG, {a}); }
 template<typename _Tp> VReg<_Tp> fma(const VReg<_Tp>& a, const VReg<_Tp>& b, const VReg<_Tp>& c)
 { return newiopV<_Tp>(VOP_FMA, {a, b, c}); }
+template<typename _Tp> VReg<_Tp> fma(const VReg<_Tp>& a, const VReg<_Tp>& b, const VReg<_Tp>& c, int64_t index)
+{ return newiopV<_Tp>(VOP_FMA, {a, b, c, index}); }
+template<typename _Tp> VReg<_Tp> ext(const VReg<_Tp>& n, const VReg<_Tp>& m, int64_t index)
+{ newiopNoret(VOP_ARM_EXT, {n, m, index}); }
```

Comment:
It must be used newiopV<_Tp> in ext, not newiopNoret, since ext creates new register.